src/lib/MessageStream.ts

```diff
@@ -146,21 +148,24 @@ export class MessageStream implements AsyncIterable<MessageStreamEvent> {
     for (const message of params.messages) {
       runner._addMessageParam(message);
     }
-    runner._run(() =>
-      runner._createMessage(
-        messages,
-        { ...params, stream: true },
-        { ...options, headers: { ...options?.headers, 'X-Stainless-Helper-Method': 'stream' } },
-      ),
-    );
+    void runner
```

Comment:
`void` has no runtime effect; it’s typically just used as an `eslint-disable` comment for `@typescript-eslint/no-floating-promises` 

in this case you don’t have a floating promise because you have a `.catch()`, so I think you should be able to remove the `void` without getting a lint error